src/main/java/io/stargate/sgv2/jsonapi/service/processor/CommandProcessor.java

```diff
@@ -50,16 +56,24 @@ public CommandProcessor(
    */
   public <T extends Command> Uni<CommandResult> processCommand(
       CommandContext commandContext, T command) {
-    // start by resolving the command, get resolver
-    return commandResolverService
-        .resolverForCommand(command)
+    // vectorize the data
```

Comment:
Vectorize will be the first step so that resolvers can use the vector value for Operation creation